tools/js2c.py

```diff
@@ -278,6 +278,9 @@ def JS2C(source, target):
     else:
       id = s
 
+    if 'node.1' in id:
+      id = id.replace('node.1', 'manual')
```

Comment:
It should probably be `if id.endswith('node.1'):` but looks fine to me apart from that.